doc/source/ray-observability/user-guides/debug-apps/optimize-performance.rst

```diff
@@ -62,19 +62,19 @@ as well as some known problems. If you encounter other problems, please
 Visualizing Tasks in the Ray Timeline
 -------------------------------------
 
-The most important tool is the timeline visualization tool. To visualize tasks
-in the Ray timeline, you can dump the timeline as a JSON file by running ``ray
-timeline`` from the command line or ``ray.timeline`` from the Python API.
+1. The :ref:`timeline API <ray-core-timeline>` is available from Ray Dashboard.
 
-To use the timeline, Ray profiling must be enabled by setting the
-``RAY_PROFILING=1`` environment variable prior to starting Ray on every machine, and ``RAY_task_events_report_interval_ms`` must be larger than 0 (default 1000).
+* Download the Chrome tracing file by clicking the download button.
+* Second, you can use tools like ``chrome://tracing`` or the `Perfetto UI <https://ui.perfetto.dev/>`_ and drop the downloaded chrome tracing file. Using the Perfetto UI is the recommended way to visualize Chrome tracing files.
+* See the timeline visualization of Ray Tasks and Actors. There are Node rows (hardware) and Worker rows (processes). Each worker row displays a list of events (e.g., Task scheduled, Task running, input/output deserialization, etc.) from that Worker over time.
 
+2. You can also export the tracing file as a JSON file by running ``ray timeline`` from the command line or ``ray.timeline`` from the Python API.
 .. code-block:: python
+ray.timeline(filename="/tmp/timeline.json")
+
+To use the timeline, enable Ray profiling by setting the ``RAY_PROFILING=1`` environment variable prior to starting Ray on every machine, and setting the ``RAY_task_events_report_interval_ms`` to larger than 0 (default 1000).
```

Comment:
Both are on by default, so we can remove it I think cc @ricky to confirm.

Reply:
Removed this for now.